# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dvbd/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dvbd-0.7.7/src/stringrep.h.vanilla	2006-03-04 16:37:18.856263250 +0100
+++ dvbd-0.7.7/src/stringrep.h	2006-03-04 16:37:31.621061000 +0100
@@ -26,7 +26,7 @@
 class StringRep {
 public:
   friend std::ostream& operator << (std::ostream &o, const StringRep &s);
-  std::string StringRep::toString() const;
+  std::string /*StringRep::*/toString() const;
 protected:
   virtual ~StringRep();
   virtual std::ostream& printOn(std::ostream &) const = 0;
--- dvbd-0.7.7/src/tunermanager.h.vanilla	2006-03-04 16:36:36.345606500 +0100
+++ dvbd-0.7.7/src/tunermanager.h	2006-03-04 16:36:49.482427500 +0100
@@ -38,7 +38,7 @@
 
   virtual ~TunerManager();
 
-  bool TunerManager::parseConfig(ConfigFile &c);
+  bool /*TunerManager::*/parseConfig(ConfigFile &c);
   void parseCommandLine(int argc, char *argv[]);
 
   virtual void addSelectFDs(Select &) const;
